### PR TITLE
[MC-70]: Port changes from public docs into provider repo README

### PR DESCRIPTION
**Pull request by sumitramanga**
Overview
----

This PR is to port the public docs changes (raygun.com/docs) into this PR.

Author to check 👓
- [ ]  Builds pass
- [ ]  Tested in an alternative environment
- [x]  Reviewed by another developer
- [x]  Appropriate documentation written (code comments, Quip docs)

Reviewer to check ✔️
- [ ]  Change has been tested on Beta
- [ ]  Code is written to standards
- [ ]  Appropriate tests have been written (code comments, Quip docs)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,43 +156,48 @@ Crash reports sent to Raygun from this Serilog sink will include HTTP context de
 
 Setting `ignoredFormFieldNames` to a list that only contains "*" will cause no form fields to be sent to Raygun. Placing * before, after or at both ends of an entry will perform an ends-with, starts-with or contains operation respectively.
 
-Note that HTTP headers, query parameters, cookies, server variables and raw request data can also be filtered out. Configuration to do so is described in the [RaygunSettings](#raygun4net-features-configured-via-raygunsettings) section further below.
+Note that HTTP headers, query parameters, cookies, server variables and raw request data can also be filtered out. Configuration to do so is described in the [RaygunSettings](https://raygun.com/documentation/language-guides/serilog-sinks/crashreporting/installation/#raygun4net-features-configured-via-raygunsettings) section further below.
```
Comment (crystalqdm) [inline README.md:159]: Do we want this to link to the raygun docs or just stay within github?

File: README.md [hunk 2]
```diff
@@ -124,43 +156,48 @@ Crash reports sent to Raygun from this Serilog sink will include HTTP context de
...
-### tagsProperty
+
+**tagsProperty**
+
 `type: string`
 
 `default: Tags`
 
-This allows you to specify a key in the properties collection that contains a list of tags to include on crash reports. Note that these will be included in addition to any global tags [described above](#tags). If you set a list of tags in the properties collection multiple times (e.g. at different logging scopes) then only the latest list of tags will be used.
+This allows you to specify a key in the properties collection that contains a list of tags to include on crash reports. Note that these will be included in addition to any global tags [described above](https://raygun.com/documentation/language-guides/serilog-sinks/crashreporting/installation/#tags). If you set a list of tags in the properties collection multiple times (e.g. at different logging scopes) then only the latest list of tags will be used.
```
Comment (crystalqdm) [inline README.md:183]: Same here about the link

**Review by crystalqdm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sumitramanga) [reply README.md]: Removed

**Review by crystalqdm — APPROVED** [review]
looks good! 🚀

**Pull request merged**